integration_testing/features/learner/learner-engage-content-channels.feature

```diff
@@ -18,13 +18,17 @@ Feature: Learner engages content channels
       And I see all the content items of the subtopic <subtopic> of the <topic> topic for the channel <channel>
     When I click the <content_item> content item
     Then I see the *Channels > '<channel>' > '<topic>' > '<subtopic>' > '<content_item>'* breadcrumb
+    When I click on the <content_item> card
+    Then I see the <content_item> page
       And I see the video start playing automatically
       # bug in Firefox, issue reported
       And I see the *Next resource* heading and content item under the player
     When video finishes
-    Then I see the *Next* snackbar alert with the title of the next recommended content item
-      And I see the *+500 points* snackbar alert
-      And I see my points counter is increased by 500 
+    Then I see the status icon is yellow star
+    # Then I see the *Next* snackbar alert with the title of the next recommended content item
```

Comment:
@khangmach Please confirm that the snackbars alerts for `next resource` (above) and points (see the comment below, line #30) were removed intentionally in 0.13

Reply:
I was not made aware of this removal. Not intentional as far as I'm aware.